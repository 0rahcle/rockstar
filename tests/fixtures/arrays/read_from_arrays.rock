Rock X with 1, 2, 3, 4, 5, 6, 7, 8, 9, 0
Until X is nothing
Roll X into Y, Z
Shout X
Shout Y with " - " with Z

Rock X with true, false, 1, 0, "yes", "no"
Shout X
Until X is nothing
Roll X into the night
Shout the night

Rock numbers with 1, 2, 3
Rock strings with "a", "b", "c"
Rock bools with true, false
Rock matrix with numbers, strings, bools
Until matrix is nothing
Roll matrix into current
Until current is nothing
Roll current into output
Shout output



