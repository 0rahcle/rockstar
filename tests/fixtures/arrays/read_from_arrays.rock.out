[ 3, 4, 5, 6, 7, 8, 9, 0 ]
1 - 2
[ 5, 6, 7, 8, 9, 0 ]
3 - 4
[ 7, 8, 9, 0 ]
5 - 6
[ 9, 0 ]
7 - 8
[]
9 - 0
[ true, false, 1, 0, 'yes', 'no' ]
true
false
1
0
yes
no
1
2
3
a
b
c
true
false
